README.md

```diff
@@ -201,6 +201,62 @@ This will match the export name `"react-server"` and `"edge-light"` then use the
 
 `bunchee` has support for checking the package bundles are matched with package exports configuration.
 
+### Wildcard Exports (Experimental)
+
+Bunchee implements the Node.js feature of using the asterisk `*` as a wildcard to match the exportable entry files.
+
+For example:
+
+```json
+{
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js"
+    },
+    "./*": {
+      "import": "./dist/*.mjs",
+      "require": "./dist/*.cjs"
+    }
+  }
+}
+```
+
+The asterisk `*`  will be replaced with your entry files. Such as:
+
+```
+- my-lib/
+  |- src/
+    |- foo/
+      |- index.ts
+    |- bar.ts
+    |- index.ts
+  |- package.json
+```
+
+This will match the export names `"foo"` and `"bar"` and will be treated as the new entries.
```

Comment:
```suggestion
This will match the export names `"foo"` and `"bar"` and will be treated as the new entries as they matched the `./*` wildcard in `my-lib` folder.
```